app/models/user.rb

```diff
@@ -0,0 +1,2 @@
+class User < ApplicationRecord
+end
```

Comment:
Please set up the associations between models. Check the milestone requirements.